uikit/src/main/java/com/midtrans/sdk/uikit/views/banktransfer/status/BankTransferStatusPresenter.java

```diff
@@ -37,8 +40,15 @@ public String getVaNumber() {
                     vaNumber = transactionResponse.getPermataVANumber();
                     break;
                 case PaymentType.ALL_VA:
-                    //VA number is based on other VA processor
-                    vaNumber = TextUtils.isEmpty(transactionResponse.getBniVaNumber()) ? transactionResponse.getPermataVANumber() : transactionResponse.getBniVaNumber();
```

Comment:
have you tested this code miss behave or not working?

Reply:
Yes, I've tested that code. That code cannot handle bri as other VA Processor, that's why I changed it.

Reply:
Ok, so it also can be test and confirmed by QA